prac_05/emails.py

```diff
@@ -0,0 +1,31 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# Dictionary of Australian state codes to their full names
+CODE_TO_NAME = {
+    "QLD": "Queensland",
+    "NSW": "New South Wales",
+    "NT": "Northern Territory",
+    "WA": "Western Australia",
+    "ACT": "Australian Capital Territory",
+    "VIC": "Victoria",
+    "TAS": "Tasmania"
+}
+
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ").upper()
+
+while state_code != "":
+    try:
```

Comment:
The error handling for invalid state codes using a try-except block is effective. You can try to add a prompt to encourage the user to enter a valid state code after an "Invalid short state" message, improving user experience.